examples_tests/17.SimpleBulletIntegration/Camera.hpp

```diff
@@ -17,27 +17,22 @@ using namespace ui;
 
```

Comment:
just move this header to some shared headers folder (you're gonna need the camera in a lot of examples)

Reply:
Is example_tests/common a good place for it?

Reply:
sure

Reply:
done